src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Run NRQL queries asynchronously"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+You can use our NerdGraph API to run an asynchronous NRQL query. This is useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. It's also especially applicable for running queries that may take a long time to complete.
+
+## Requirements [#requirements]
+
+The normal [NRQL query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries) is one minute. To run queries with a 10-minute limit, you must have [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-plus). (edited) 
+
+## Asynchronous query example [#example]
+
+Here's an example of an asynchronous query NerdGraph call. The `async: true` is what makes the query asynchronous. This query will return results if it completes within the default timeout of 5 seconds; otherwise it will return `queryProgress` data for use in follow-up queries to the `actor.account.nrqlQueryProgress` field. 
+
+```
+{
+   actor {
+      account(id: <var>YOUR_ACCOUNT_ID</var>) {
+         nrql(query: <var>"SELECT * FROM Transaction"</var>, async: true) {
+            results
+            queryProgress {
+               queryId
+               completed
+               retryAfter
+               retryDeadline
+               resultExpiration
+            }
+         }
+      }
+   }
+}
+```
+
+## Getting the results or status of an async query [#example-query-results]
+
+When you run the previous query, it will return a query ID, which you can use to get results or progress status for that query. 
```

Comment:
```suggestion
If your query exceeds the default timeout, it will return a query ID. Take that ID and run a second query to retrieve the results or progress status for that query. 
```